packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -529,4 +530,32 @@ export class MetriportMedicalApi {
     const resp = await this.api.post(url, docRef);
     return resp.data;
   }
+
+  /**
+   * Verifies the signature of a webhook request.
+   * Refer to Metriport's documentation for more details: https://docs.metriport.com/medical-api/more-info/webhooks.
+   *
+   * @param wh_key - your webhook key
+   * @param cxId - your cxID
+   * @param req.body - the body of the webhook request
+   * @param timestamp - tthe timestamp obtained from the webhook request header
+   * @param signature - the signature obtained from the webhook request header
+   *
+   * @returns True if the signature is verified, false otherwise
+   */
+  verifyWebhookSignature = (
+    wh_key: string,
+    reqBody: string,
+    signature: string | string[] | undefined
```

Comment:
The type when you unpack the req.headers["x-metriport-signature"] is a string | string[] | undefined
So this is a little weird since they can pass in an undefined and then we through in an error at runtime instead of compile time, but other than validating it is a string before calling the function (which is what I had before), I can't think of another way to do this. 
Thoughts?

Reply:
I think we should require it to be string _and_ also convert it to string using `String(signature)`, so even if they pass something that's not a string it will not break the function - it just doesn't validate.